docs/topics/api/reviews.rst

```diff
@@ -8,17 +8,21 @@ Reviews
     may change without warning. The only authentication method available at
     the moment is :ref:`the internal one<api-auth-internal>`.
 
--------------------
-List Add-on reviews
--------------------
+------------
+List reviews
+------------
 
-.. review-list-addon:
+.. review-list:
 
-This endpoint allows you to fetch reviews for a given add-on.
+This endpoint allows you to fetch reviews for a given add-on or user. Either
+``addon`` or ``user`` query parameters are required, and they can be
+combined together.
 
-.. http:get:: /api/v3/addons/addon/(int:id|string:slug|string:guid)/reviews/
+.. http:get:: /api/v3/reviews/review/(int:id|string:slug|string:guid)/reviews/
```

Comment:
Good catch, this is wrong. The actual URL is just `/api/v3/reviews/review/`, I'll fix